src/Controllers/UserController.js

```diff
@@ -60,11 +60,18 @@ export class UserController extends AdaptableController {
       updateFields._email_verify_token_expires_at = {__op: 'Delete'};
     }
 
-    return this.config.database.update('_User', query, updateFields).then((document) => {
-      if (!document) {
-        throw undefined;
-      }
-      return Promise.resolve(document);
+    var self = this;
+    var checkIfAlreadyVerified = new RestQuery(self.config, Auth.master(self.config), '_User', {username: username, emailVerified: true});
```

Comment:
But when the email field is changed on a _User, emailVerified gets set to false. So this wouldn't affect the ability to verify a changed email.

Also, removing this isn't _that_ big a deal to me, it's only here so that if you click a link and have already verified your email that you send up on the `verify_email_success` page instead of the invalid_verification_link page, because that feels misleading.